.cron.yml

```diff
@@ -10,7 +10,6 @@ jobs:
           treeherder-symbol: Nd
           target-tasks-method: nightly
       when:
-          - {hour: 6, minute: 0}
```

Comment:
Removing this line avoid scheduling the same jobs at the same time. `nightly-on-google-play` schedules exactly the same jobs as `nightly` + the task that publishes to GP.